src/metabase/driver/sql/query_processor/deprecated.clj

```diff
@@ -0,0 +1,128 @@
+(ns metabase.driver.sql.query-processor.deprecated
+  "Deprecated stuff that used to live in [[metabase.driver.sql.query-processor]]. Moved here so it can live out its last
+  days in a place we don't have to look at it, and to discourage people from using it. Also convenient for seeing
+  everything that's deprecated at a glance.
+
+  Deprecated method impls should call [[log-deprecation-warning]] to gently nudge driver authors to stop using this
+  method."
+  (:require [clojure.tools.logging :as log]
+            [metabase.driver :as driver]
+            [metabase.query-processor.store :as qp.store]
+            [metabase.query-processor.util.add-alias-info :as add]
+            [metabase.util :as u]
+            [metabase.util.i18n :refer [trs]]))
+
+;; TODO -- this is actually pretty handy and I think we ought to use it for all the deprecated driver methods.
```

Comment:
why not every deprecated method anywhere?

Reply:
Yeah we could use it in a bunch of places for sure. I didn't want the PR to get too off track by doing here tho